### PR TITLE
Update `eslint-airbnb`.

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**

### CODE REVIEW

File: tests/connectModule/singleModule-test.js [hunk 1]
```diff
@@ -33,7 +36,7 @@ describe('ConnectedComponent', () => {
         <div>
           <Single />
         </div>
-      </ModuleProvider>
+      </ModuleProvider>,
```
Comment (mboperator) [inline tests/connectModule/singleModule-test.js]: I don't know how I feel about the trailing comma on JSX blocks in parens.

File: src/moduleProvider/index.js [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+import { Children, Component, PropTypes } from 'react';
 import { combineReducers } from 'redux';
 
 import registerModule from './registerModule';
 
 import storeShape from '../utils/storeShape';
 
 export default class ModuleProvider extends Component {
-  static propTypes = {
+  static propTypes ={
```
Comment (mboperator) [inline src/moduleProvider/index.js]: The updated lint rules don't accept a space after the `=` in this case?

Comment (wtgtybhertgeghgtwtg) [inline src/moduleProvider/index.js]: I think that was just `esformatter` being overeager.  I'll fix that.

**Comment by mboperator** [comment]
I'll merge this one once conflicts are resolved, @wtgtybhertgeghgtwtg

**Comment by wtgtybhertgeghgtwtg** [comment]
Conflicts resolved.

**Comment by mboperator** [comment]
Fantastic, thank you man.

**Pull request merged**